core/src/message/handle.rs

```diff
@@ -26,6 +26,8 @@ use codec::{Decode, Encode};
 use scale_info::TypeInfo;
 
 /// Handle message.
+/// 
+/// This message subclass specialized to be used in context::MessageContext::send_commit method
```

Comment:
A dot on the end of the sentence.
```suggestion
/// This message subclass specialized to be used in context::MessageContext::send_commit method.
```

Reply:
done